edu.stanford.Almond.json

```diff
@@ -0,0 +1,272 @@
+{
+    "app-id" : "edu.stanford.Almond",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.gfortran-62"
+    ],
+    "command" : "/app/bin/edu.stanford.Almond",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=host",
+        "--talk-name=ca.desrt.dconf",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.login1",
```

Comment:
Is it actually needed?

Reply:
One of the supported commands is "lock my pc", which is implemented by calling out to logind.